src/packet/secret_key.js

```diff
@@ -175,7 +175,7 @@ SecretKey.prototype.write = function () {
  * This can be used to remove passphrase protection after calling decrypt().
  * @param {String} passphrase
  */
-SecretKey.prototype.encrypt = function (passphrase) {
+SecretKey.prototype.encrypt = function (passphrase, options) {
```

Comment:
just using a second `count` parameter here is fine. No need to use an options object.